build.sbt

```diff
@@ -30,12 +35,18 @@ val dep = {
 val Integration = config("it") extend Test
 
 val root = project.in(file("."))
-    .configs(Integration)
-    .settings(commonSettings: _*)
-    .settings(libraryDependencies ++= dep)
-    .settings(inConfig(Integration)(Defaults.testSettings) : _*)
+  .configs(Integration)
+  .settings(commonSettings: _*)
+  .settings(libraryDependencies ++= dep)
+  .settings(inConfig(Integration)(Defaults.testSettings) : _*)
 
 scalacOptions := Seq("-unchecked", "-deprecation", "-feature")
 
 (scalastyleConfig in Test) := baseDirectory.value / "scalastyle-test-config.xml"
-scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
\ No newline at end of file
+scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
+
+mappings in Universal ++= (resourceDirectory in Compile).value.listFiles()
```

Comment:
It's true, it sucks to have to remember to put new config in both places. The theory is that production settings will be different and more stable than development settings. Furthermore if there's a `resources.conf` that config file will end up embedded in the jar and be used first. So we could have common config in a single `resources.conf` and only have config that is different between production and development in `application.conf`. Might be worth investigating in a later task.